### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by callumreid**
-Made editProfile inputs track value in state and psuedo-submit on button press
-Made changePassword fields have to match (very basic but a start on validation)
-Created fake data and passed it in to profile component; should be a reasonable approximation to how the profile will handle data from an API
-Deleted some unnecessary files
-Did some setup for creating a dakmode. Currently just blacks out everything but is responsive to toggle
-Reorganized all my screens into a profile folder for modularity's sake
-Merged with most recent changes. Currently have the login page toggled to 'already logged in' to avoid bug in google auth. Can be undone by setting the `isLoggedIn` useToggle to false

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,42 +1,70 @@
...
+      ...PaperDefaultTheme.colors
+    }
+  };
+
+  const customDarkTheme = {
+    ...NavigationDarkTheme,
+    ...PaperDarkTheme,
+    colors: {
+      ...NavigationDarkTheme.colors,
+      ...PaperDarkTheme.colors
```
Comment (thegraceoflayla) [inline App.js:59]: I've never seen this syntax before, but I like how clean it is!

File: data/profileData.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const fakeUser =  {
```
Comment (thegraceoflayla) [inline data/profileData.js:1]: I appreciate you keeping this as similar as possible to the defined API response.

**Review by thegraceoflayla — APPROVED** [review]
All in all, this looks great! Seems like you've done a good job planning ahead for connecting to the backend. Once that's done, try to add some tests, if you can.

File: App.js (hunk 1 above)
Comment (callumreid) [reply App.js:59]: Yea, the spread syntax is something I am still not totally comfortable with but trying to get more used to it

**Pull request merged**